src/std/srfi/141.ss

```diff
@@ -3,6 +3,7 @@
 ;;; SRFI-141: Integer Division
 
 (import :gerbil/gambit
+        :scheme/base
```

Comment:
No.

Circular dependency.

stdlib cannot and should noy use anything from `:scheme/`

Reply:
Fixed.